ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -12,6 +12,7 @@
     - '{{ docker_image_prefix }}/python2action'
     - '{{ docker_image_prefix }}/python3action'
     - '{{ docker_image_prefix }}/swift3action'
+    - '{{ docker_image_prefix }}/action-swift-v3.1.1'
```

Comment:
@akrabat We are trying to come up with a better convention for the docker images, we are going to have a lot of images under the same prefix/account locally in dockerhub, so having the `action` easier to identify the action images, and the version at the end helps for sorting, as we add more kinds base on new versions of the language goin forward